setup.cfg

```diff
@@ -0,0 +1,19 @@
+[metadata]
+name = eligibility
+version = 2022.01.1
+description = Data exchange to verify eligibility for transit benefits
+url = https://github.com/cal-itp/eligibility-api
+project_urls =
+    Bug Tracker = https://github.com/cal-itp/eligibility-api/issues
+license = 'Apache 2.0'
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+
+[options]
+packages = find:
+python_requires = >=3.9
+install_requires =
+    jwcrypto==1.0
+    requests==2.27.1
```

Comment:
Prompted by @thekaveman 's feedback, I found [this Python docs page](https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/) talking about `install_requires` vs `requirements.txt`.

A few sections that confirm we shouldn't pin to specific versions in `setup.cfg`:

> It is not considered best practice to use `install_requires` to pin dependencies to specific versions, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades.

> Whereas `install_requires` requirements are minimal, requirements files often contain an exhaustive listing of pinned versions for the purpose of achieving repeatable installations of a complete environment.